frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -2088,12 +2097,12 @@ mod tests {
 
 			let (solution, _) = MultiPhase::mine_solution::<<Runtime as Config>::Solver>().unwrap();
```

Comment:
would be good if 
```
primitives/npos-elections/src/lib.rs|149| 10:pub type ElectionScore = [ExtendedBalance; 3];
```
also switch to the new struct.

Reply:
same with https://github.com/paritytech/substrate/pull/10834#discussion_r805736070